Java/java_substring_comparisons

```diff
@@ -0,0 +1,43 @@
+QUESTION LINK: https://www.hackerrank.com/challenges/java-string-compare/problem
```

Comment:
QUESTION
We define the following terms: Lexicographical Order, also known as alphabetic or dictionary order, orders characters as follows: A<B<C....<Y<Z<a<b...….<y<z
For example, ball < cat, dog < dorm, Happy < happy, Zoo < ball.
A substring of a string is a contiguous block of characters in the string. For example, the substrings of abc are a, b, c, ab, bc, and abc.
Given a string,s, and an integer,k , complete the function so that it finds the lexicographically smallest and largest substrings of length k .
Input Format:
The first line contains a string denoting s .
The second line contains an integer denoting k .
Constraints:
1<=|s|<=1000
s consists of English alphabetic letters only 
Output Format
Return the respective lexicographically smallest and largest substrings as a single newline-separated string
Sample Input 0
welcometojava
3
Sample Output 0
ava
wel